symfony/plugins/orangehrmMarketPlacePlugin/modules/marketPlace/templates/ohrmAddonsSuccess.php

```diff
@@ -78,7 +78,7 @@
                                            echo 'background-color: #808080;';
```

Comment:
Seems hardcoded style here

Reply:
It is depending on the addonl list pass to success file. I can create class and add PHP to make it available when if condition is there. Is it ok?

Reply:
corrected